src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -110,27 +110,25 @@ export default class ArcanaTracking extends Module {
 
 		this.addHook('removebuff', {abilityId: DRAWN_ARCANA, by: 'player'}, this.offDrawnStatus)
 		this.addHook('death', {to: 'player'}, this.onDeath)
-		this.addHook('complete', this.onComplete)
 	}
 
-	normalise(events: any) {
+	normalise(events: Event[]) {
 		const startTime = this.parser.fight.start_time
 		let prepullSleeve = false
 		const sleeveDrawLog: CastEvent[] = []
-
 		for (const event of events) {
 			if ((event.timestamp > startTime && !prepullSleeve)
 				|| event.timestamp - startTime >= (STATUSES.SLEEVE_DRAW.duration * 1000)
 				) {
-				// End loop if: 1. Max duration of sleeve draw status passed 2. No prepull sleeve detected
-				break
+					// End loop if: 1. Max duration of sleeve draw status passed 2. No prepull sleeve detected
+					break
 
-				} else if (!prepullSleeve && event.type === 'applybuff' && STATUSES.SLEEVE_DRAW.id === event.ability.guid
+				} else if (!prepullSleeve && event.type === 'applybuff' && this.isBuffEvent(event) && STATUSES.SLEEVE_DRAW.id === event.ability.guid
```

Comment:
nit; you can include the `event.type` check in the type guard, so you don't need to include both in the conditional